website/src/pages/docs/handlers/new-openapi.mdx

```diff
@@ -0,0 +1,208 @@
+import { PackageCmd } from '@theguild/components'
+
+# New OpenAPI / Swagger
+
+![image](https://user-images.githubusercontent.com/20847995/79218686-7ba7b900-7e59-11ea-8a5e-676a83b9f86e.png)
+
+This handler allows you to load remote or local [OpenAPI (2/3) and Swagger](https://swagger.io) schemas.
+
+You can import it using remote/local `.json` or `.yaml`.
+
+To get started, install the handler library:
+
+<PackageCmd packages={['@graphql-mesh/new-openapi']} />
+
+Now, you can use it directly in your Mesh config file:
+
+```yaml
+sources:
+  - name: MyOpenapiApi
+    handler:
+      openapi:
+        source: ./my-schema.json
```

Comment:
This also can take URLs.

Reply:
you're right.
and also, `source` is replaced with `oasFilePath` :see_no_evil:

Reply:
haha I also missed that :D